Paraswap-Agents/AdminOperations/src/agent.spec.ts

```diff
@@ -0,0 +1,243 @@
+import { BigNumber } from "ethers";
+import { Interface } from "ethers/lib/utils";
+import { Finding, FindingType, FindingSeverity, TransactionEvent, HandleTransaction } from "forta-agent";
+import { createAddress, TestTransactionEvent } from "forta-agent-tools/lib/tests";
+import { provideHandleTransaction } from "./agent";
+import { SWAPPER_IFACE } from "./utils";
+
+// AugustusSwapper address
+const SWAPPER_ADDR = createAddress("0xa1");
+// Address used to initiate transactions
+const USER_ADDR = createAddress("0xaa");
+// Irrelevent events/ functions used for tests
```

Comment:
Really not a big deal, but there are some typos of "irrelevent", which should probably be "irrelevant".